dashboard/modules/metrics/dashboards/serve_grafana_dashboard_base.json

```diff
@@ -52,17 +52,17 @@
           ]
         },
         "datasource": "${datasource}",
-        "definition": "label_values(ray_serve_num_http_requests{{{global_filters}}}, route)",
+        "definition": "label_values({{__name__=~ 'ray_serve_num_http_requests|ray_serve_num_grpc_requests',{global_filters}}}, application)",
         "description": null,
         "error": null,
         "hide": 0,
         "includeAll": true,
         "label": null,
         "multi": true,
-        "name": "Route",
+        "name": "Application",
         "options": [],
         "query": {
-          "query": "label_values(ray_serve_num_http_requests{{{global_filters}}}, route)",
+          "query": "label_values({{__name__=~ 'ray_serve_num_http_requests|ray_serve_num_grpc_requests',{global_filters}}}, application)",
```

Comment:
Should we change this to `ray_serve_deployment_replica_healthy` (like what we use in `serve_deployment_grafana_dashboard_base.json`)? If no requests has been sent yet, might the application not show up in the dropdown?

Reply:
oh that's great call out! Let me update it